File: app/src/main/java/cl/awakelabs/ejercicio4/MainActivity.java
package cl.awakelabs.ejercicio4;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button suma = findViewById(R.id.btnSum);
        Button resta = findViewById(R.id.btnRest);
        Button multiplica = findViewById(R.id.btnMulti);
        Button divide = findViewById(R.id.btnDiv);

        EditText nUno = findViewById(R.id.numUno);
        EditText nDos = findViewById(R.id.numDos);

        suma.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                if (nUno.getText().toString().isBlank()) {
                    Toast.makeText(getBaseContext(), "Falta algun valor", Toast.LENGTH_SHORT).show();
                    return;
                } else if (nDos.getText().toString().isBlank()) {
                    Toast.makeText(getBaseContext(), "Falta algun valor", Toast.LENGTH_SHORT).show();
                    return;
                }

                Integer uno = Integer.parseInt(nUno.getText().toString());
                Integer dos = Integer.parseInt(nDos.getText().toString());

                Integer sumar = uno + dos;

                Toast.makeText(getBaseContext(), "Resultado: "+ sumar, Toast.LENGTH_SHORT).show();
            }
        });

        resta.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                if (nUno.getText().toString().isBlank()) {
                    Toast.makeText(getBaseContext(), "Falta algun valor", Toast.LENGTH_SHORT).show();
                    return;
                } else if (nDos.getText().toString().isBlank()) {
                    Toast.makeText(getBaseContext(), "Falta algun valor", Toast.LENGTH_LONG).show();
                    return;
                }
                Integer uno = Integer.parseInt(nUno.getText().toString());
                Integer dos = Integer.parseInt(nDos.getText().toString());

                Integer restar = uno - dos;

                Toast.makeText(getBaseContext(), "Resultado: "+ restar, Toast.LENGTH_LONG).show();

            }
        });

        multiplica.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                if (nUno.getText().toString().isBlank()) {
                    Toast.makeText(getBaseContext(), "Falta algun valor", Toast.LENGTH_SHORT).show();
                    return;
                } else if (nDos.getText().toString().isBlank()) {
                    Toast.makeText(getBaseContext(), "Falta algun valor", Toast.LENGTH_LONG).show();
                    return;
                }
                Integer uno = Integer.parseInt(nUno.getText().toString());
                Integer dos = Integer.parseInt(nDos.getText().toString());

                Integer multiplicar = uno * dos;

                Toast.makeText(getBaseContext(), "Resultado: "+ multiplicar, Toast.LENGTH_LONG).show();


            }
        });

        divide.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                if (nUno.getText().toString().isBlank()) {
                    Toast.makeText(getBaseContext(), "Falta algun valor", Toast.LENGTH_SHORT).show();
                    return;
                } else if (nDos.getText().toString().isBlank()) {
                    Toast.makeText(getBaseContext(), "Falta algun valor", Toast.LENGTH_LONG).show();
                    return;
                }
                Integer uno = Integer.parseInt(nUno.getText().toString());
                Integer dos = Integer.parseInt(nDos.getText().toString());

                Integer dividir = uno / dos;

                Toast.makeText(getBaseContext(), "Resultado: "+ dividir, Toast.LENGTH_LONG).show();

            }
        });

    }
}

File: app/src/main/java/cl/awakelabs/ejercicio4/MainActivity.kt
package cl.awakelabs.ejercicio4

import android.os.Bundle
import android.view.View
import android.widget.Button
import android.widget.EditText
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val suma = findViewById<Button>(R.id.btnSum)
        val resta = findViewById<Button>(R.id.btnRest)
        val multiplica = findViewById<Button>(R.id.btnMulti)
        val divide = findViewById<Button>(R.id.btnDiv)
        val nUno = findViewById<EditText>(R.id.numUno)
        val nDos = findViewById<EditText>(R.id.numDos)
        suma.setOnClickListener(View.OnClickListener {
            if (nUno.text.toString().isBlank()) {
                Toast.makeText(baseContext, "Falta algun valor", Toast.LENGTH_SHORT).show()
                return@OnClickListener
            } else if (nDos.text.toString().isBlank()) {
                Toast.makeText(baseContext, "Falta algun valor", Toast.LENGTH_SHORT).show()
                return@OnClickListener
            }
            val uno = nUno.text.toString().toInt()
            val dos = nDos.text.toString().toInt()
            val sumar = uno + dos
            Toast.makeText(baseContext, "Resultado: $sumar", Toast.LENGTH_SHORT).show()
        })
        resta.setOnClickListener(View.OnClickListener {
            if (nUno.text.toString().isBlank()) {
                Toast.makeText(baseContext, "Falta algun valor", Toast.LENGTH_SHORT).show()
                return@OnClickListener
            } else if (nDos.text.toString().isBlank()) {
                Toast.makeText(baseContext, "Falta algun valor", Toast.LENGTH_LONG).show()
                return@OnClickListener
            }
            val uno = nUno.text.toString().toInt()
            val dos = nDos.text.toString().toInt()
            val restar = uno - dos
            Toast.makeText(baseContext, "Resultado: $restar", Toast.LENGTH_LONG).show()
        })
        multiplica.setOnClickListener(View.OnClickListener {
            if (nUno.text.toString().isBlank()) {
                Toast.makeText(baseContext, "Falta algun valor", Toast.LENGTH_SHORT).show()
                return@OnClickListener
            } else if (nDos.text.toString().isBlank()) {
                Toast.makeText(baseContext, "Falta algun valor", Toast.LENGTH_LONG).show()
                return@OnClickListener
            }
            val uno = nUno.text.toString().toInt()
            val dos = nDos.text.toString().toInt()
            val multiplicar = uno * dos
            Toast.makeText(baseContext, "Resultado: $multiplicar", Toast.LENGTH_LONG).show()
        })
        divide.setOnClickListener(View.OnClickListener {
            if (nUno.text.toString().isBlank()) {
                Toast.makeText(baseContext, "Falta algun valor", Toast.LENGTH_SHORT).show()
                return@OnClickListener
            } else if (nDos.text.toString().isBlank()) {
                Toast.makeText(baseContext, "Falta algun valor", Toast.LENGTH_LONG).show()
                return@OnClickListener
            }
            val uno = nUno.text.toString().toInt()
            val dos = nDos.text.toString().toInt()
            val dividir = uno / dos
            Toast.makeText(baseContext, "Resultado: $dividir", Toast.LENGTH_LONG).show()
        })
    }
}